docs/citations.md

```diff
@@ -39,6 +39,16 @@ For DOIs with multiple slashes in the identifier you also have to use the full h
 
 :::
 
+### Writing DOIs to BibTeX
+
+If you encounter problems fetching DOIs from `https://doi.org`, for example the downloaded citation does not include all the data you expect or requests to `https://doi.org` are failing on an automated continuous integration platform, you may write your DOI citations to file using the MyST command
+
+```bash
+myst build --doi-bib
+```
+
+This will generate a BibTeX file `myst.doi.bib` which you may then rename, edit, and save to your project. On subsequent builds, the DOIs will load be loaded from this file rather than fetched remotely.
```

Comment:
```suggestion
This will generate a BibTeX file `myst.doi.bib` which you may then rename, edit, and save to your project. On subsequent builds, the DOIs will be loaded from this file rather than fetched remotely.
```